packages/polyfills/types/theme-global.d.cts

```diff
@@ -36,6 +36,7 @@ declare module '@mui/material/styles/createPalette.d' {
             publicThirdMain: string
             publicBg: string
             borderSecondary: string
+            mask: string
```

Comment:
CC @nuanyang233

Reply:
Why do we need this color? It doesn't seem to be a commonly used color.